transactions/src/docs/asciidoc/user/xa.adoc

```diff
@@ -0,0 +1,207 @@
+include::common.adoc[]
```

Comment:
Same answer as below: I can move the XA doc to the docs module if you believe it makes more sense.